gradle.properties

```diff
@@ -1,13 +1,13 @@
 
 # Terracotta third parties
 offheapVersion = 2.4.0
-statisticVersion = 2.0.2
+statisticVersion = 2.0-SNAPSHOT
 jcacheVersion = 1.1.0
 slf4jVersion = 1.7.25
 sizeofVersion = 0.3.0
 
 # Terracotta clustered
-terracottaPlatformVersion = 5.4.0-pre15
+terracottaPlatformVersion = 5.4-SNAPSHOT
```

Comment:
Need a new release after https://github.com/Terracotta-OSS/terracotta-platform/pull/432 is merged